egs/ami/s5b/local/rnnlm/tuning/run_lstm_tdnn_b.sh

```diff
@@ -91,9 +94,10 @@ if [ $stage -le 2 ]; then
 fi
 
 if [ $stage -le 3 ]; then
+  backstitch_opt="--backstitch-scale $alpha --backstitch-interval $back_interval"
```

Comment:
there should probably be a way to easily turn off the backstitch? e.g. a --use-backstitch false option for the script, and if set to false, backstitch_opt="" ?

Reply:
Regarding turning backstitch off: I prefer to have this level of script not be too configurable.  But I think it would be good idea to rename the script to run_lstm_tdnn_bs_1a.sh and have the link from run_lstm_tdnn_bs.sh, to encode that it's a script with backstitch.